client/my-sites/stats/wordads/highlights-section.jsx

```diff
@@ -0,0 +1,153 @@
+import { Gridicon, Popover } from '@automattic/components';
+import { Icon, starEmpty } from '@wordpress/icons';
+import { numberFormat, translate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { getWordAdsEarnings } from 'calypso/state/wordads/earnings/selectors';
+import HighlightCardSimple from './highlight-card-simple';
+
+// TODO: HighlightCard does not accept string values.
+// Should refactor to accept strings and move the business logic into the callers.
+// Then refactor this Comp to use HighlightCard again.
+
+function getAmountAsFormattedString( amount ) {
+	// Takes a Number, formats it to 2 decimal places, and prepends a "$".
+	// This mimics the existing behaviour. I'm assuming we only view/report
+	// on earnings in USD.
+	const formattedAmount = '$' + numberFormat( amount, 2 );
```

Comment:
If we need a new number formatter, would you be able to [extend those](https://github.com/Automattic/wp-calypso/tree/trunk/packages/components/src/number-formatters) so we'd have them centralised?

Reply:
I'm unsure about this suggestion. The `numberFormat()` used here is part of `i18n-calypso` (the same place our `translate()` call lives) and it's used extensively around Calypso. Does it make sense to introduce a new version of this or maybe to move our new formatters into `i18n-calypso`?

Reply:
That's a very good point. From what I can see, `numberFormat` with two parameters only formats a number but doesn't take locale into account, which means that for countries using a comma instead of a period before fractions, this won't be swapped. On the other hand, we hard-coded the dollar sign. Can this page show a different currency, or is it always USD?
Does showing a dollar value mean we use the US number format?
All the other highlight sections take locale into account (they use browser locale), so I lean towards using a localised number even with a foreign currency.
The table values make the same mistake, so if we swap, this wouldn't be constant until table numbers are also addressed.
This is not a blocking issue, so I can accept both outcomes, but we should probably create a task to look at this if we don't do it here.

Reply:
My assumption was that WordAds only deals in USD amounts and my initial focus was to replicate the existing behaviour so as not to confuse things. The "$" symbol is hard-coded in the previous implementation as well.

My preference would be to address changes in the currency handling in a separate PR though, once we've had a chance to investigate and confirm that results should always be shown in USD (or not).

I'll spin up an issue to track this and I'll make a note that the behaviour should be consistent between the highlight cards and the values in the tables.

Reply:
Opened an issues for this [here](https://github.com/Automattic/wp-calypso/issues/72492).

Reply:
Great, thanks @a8ck3n.